src/basic/cgroup-util.c

```diff
@@ -65,7 +65,7 @@ int cg_cgroupid_open(int cgroupfs_fd, uint64_t id) {
         cg_file_handle fh = CG_FILE_HANDLE_INIT;
         CG_FILE_HANDLE_CGROUPID(fh) = id;
 
-        int fd = open_by_handle_at(cgroupfs_fd, &fh.file_handle, O_DIRECTORY|O_CLOEXEC);
+        int fd = open_by_handle_at(cgroupfs_fd, &fh.file_handle, O_DIRECTORY|O_CLOEXEC|O_PATH);
```

Comment:
This makes things inconsistent with cg_path_open

Reply:
BTW, if you touch this, please simplify the code by using `return RET_NERRNO(open_by_handle_at(...));`